src/sentry/seer/signed_seer_api.py

```diff
@@ -33,35 +36,25 @@ def make_signed_seer_api_request(
     with metrics.timer(
         "seer.request_to_seer",
         sample_rate=1.0,
-        # Pull off query params, if any
         tags={"endpoint": parsed.path},
     ):
         return connection_pool.urlopen(
             "POST",
-            parsed.path + "?" + parsed.query,
+            parsed.path,
             body=body,
             headers={"content-type": "application/json;charset=utf-8", **auth_headers},
             **timeout_options,
         )
 
 
-def get_seer_salted_url(url: str) -> tuple[str, str]:
-    if random() < options.get("seer.api.use-nonce-signature"):
-        salt = uuid4().hex
-        url += "?nonce=" + salt
-    else:
-        salt = url
-    return url, salt
-
-
-def sign_with_seer_secret(salt: str, body: bytes):
+def sign_with_seer_secret(body: bytes) -> dict[str, str]:
     auth_headers: dict[str, str] = {}
     if random() < options.get("seer.api.use-shared-secret"):
```

Comment:
Just a reminder for us to come back and remove this. The value is set to 1.0 at the moment so it'll always apply.